Sources/Basics/HTTPClient.swift

```diff
@@ -20,9 +20,7 @@ import TSCBasic
 
 #if canImport(Glibc)
 import Glibc
-#endif
-
-#if os(Windows)
+#elseif canImport(CRT)
```

Comment:
Is there a risk of something called CRT in the search path on other platforms?

Reply:
yes, but I dont think its very realistic. we can always make it more complicated if/when we run into that